mods/default/chests.lua

```diff
@@ -129,17 +132,22 @@ function default.chest.register_chest(prefixed_name, d)
 				return itemstack
 			end
 
+			local cn = clicker:get_player_name()
+
+			if default.chest.open_chests[cn] then
+				default.chest.chest_lid_close(cn)
+			end
```

Comment:
Couldn't this check be added directly to `chest_lid_close`? It's repeated even more often now.

Reply:
> Couldn't this check be added directly to `chest_lid_close`?

you mean, check whether the chest is open from within chest_lid_close? i'm mostly following existing style w/ this PR, and i'm not sure there's a meaningful performance-related reason to refactor this check, though perhaps there's a reason to do so for legibility. 

one related change would be to have `local chest = default.chest` at the file-level scope, which would save a map lookup on every reference.

but perhaps such minor optimizations should be part of another issue?

Reply:
Well whatever. It does not matter. Looks good nonetheless.